apis/user/v1alpha1/group_types.go

```diff
@@ -31,11 +31,6 @@ type GroupParameters struct {
 	//+optional
 	//+kubebuilder:default=true
 	Active bool `json:"active"`
-	// GroupID is the group ID (known as Name in the GUI).
-	//+kubebuilder:validation:MinLength=1
-	//+kubebuilder:validation:MaxLength=64
-	//+kubebuilder:validation:Pattern=`^[A-Za-z0-9_-]*$`
-	GroupID string `json:"groupId"`
```

Comment:
What's the convention for using externalname rather than making it part of spec? Will no longer validate the id when using externalname.

Reply:
Interesting discussion https://github.com/crossplane/crossplane issue 1863 (did not want to crossref)